src/utils/date.ts

```diff
@@ -10,4 +37,125 @@ export function isDate(dateString?: ValueType) {
   }
 
   return !isNaN(new Date(dateString).getDate());
-}
\ No newline at end of file
+}
+
+export enum DATE_FORMATS {
+  TIME = "h:mm A",
+  SHORT = "MMM DD",
+  LONG = "h:mm A, MMM DD, YYYY",
+  TIME_RANGE = "YYYY-MM-DD HH:mm"
+}
+
+type formatDateOptions = {
```

Comment:
These options are not intuitive, rather create dedicated functions

Reply:
created dedicated functions as suggested.

Reply:
Don't export the formatDate

Reply:
Done